src/components/common/NavBar.js

```diff
@@ -2,24 +2,47 @@ import React from 'react';
 
 /**
  * Required Props:
- * brandTitle {string} - String representing the website brand name
- * TODO: Include other nav bar links
+ * navBarType {String} - String representing which version of the navbar to render
+ *    Nav Bar Types: 'module', 'homepage', 'catalog', and (??)
+ * 
+ * TODO: Include props for other nav bar information/links
  */
 function NavBar(props) {
-  const { brandTitle } = props;
+  const { navBarType } = props;
+
+  let navBackBtn = undefined;
```

Comment:
We can probably remove the "nav" prefix on all these variables. Its assumed by being local to NavBar.